config/build.documentation.ts

```diff
@@ -34,11 +34,12 @@ export async function buildReferenceDocSnippetExtracts(
 
     await rmRf('snippet-extractor-output');
     await mkDir('snippet-extractor-output');
-    await Promise.all(snippetExtractsPerHost.map((extracts, index) => {
-        let contents = jsyaml.safeDump(extracts);
-        let fileName = `snippet-extractor-output/${files[index].substr(0, '.xlsx'.length)}.yaml`;
-        return writeFile(path.resolve(fileName), contents);
-    }));
+
+    let contents = '';
+    snippetExtractsPerHost.map((extracts, index) => {
```

Comment:
I'm kinda amused by your use of `map` as a `forEach`
Might I instead suggest:

const contents = snippetExtractsPerHost.map(extracts => jsyaml.safeDump(extracts))
    .join("");

Reply:
I suggest your replace it with my code above.  Then feel free to squash and merge.